lib/analytics/course_students_csv_builder.rb

```diff
@@ -42,9 +46,19 @@ def row(courses_user)
     row << courses_user.character_sum_draft
     row << courses_user.references_count
     row << newbie?(courses_user.user)
+    row << total_articles_created(courses_user.user_id)
+    row << total_articles_edited(courses_user.user_id)
   end
 
   def newbie?(user)
     (@course.start..@course.end).cover? user.registered_at
   end
+
+  def total_articles_created(user_id)
+    @created_articles.count {|a| a[:user_ids].include?(user_id)}
```

Comment:
This needs to account for a situation where a newly created article was edited by multiple users in the course. In that situation, only the user who actually created it should have it counted as a created article, while everyone who edited it should have it counted as an edited article.